features/ui/button/button.tsx

```diff
@@ -1,9 +1,80 @@
 import { ButtonHTMLAttributes } from "react";
 import classNames from "classnames";
 import styles from "./button.module.scss";
+import Image from "next/image";
 
-export function Button(props: ButtonHTMLAttributes<HTMLButtonElement>) {
+export enum Size {
+  small = "small",
+  medium = "medium",
+  large = "large",
+  xlarge = "xlarge",
+}
+
+export enum Color {
+  primary = "primary",
+  secondary = "secondary",
+  gray = "gray",
+  empty = "empty",
+  emptyGray = "emptyGray",
+  error = "error",
+}
+
+export interface ButtonIcon {
+  src: string;
+  alt: string;
+  width: number;
+  height: number;
+}
+
+interface ButtonProps extends ButtonHTMLAttributes<HTMLButtonElement> {
+  size?: Size;
+  color?: Color;
+  isDisabled?: boolean;
```

Comment:
It would make sense to use `disabled` as name here as it's already part of the native `button` element.

Reply:
👍